modules/distribution/src/assembly/bin.xml

```diff
@@ -502,6 +502,18 @@
             </excludes>
         </fileSet>
 
+        <!-- copying CarbonSaaS Sample App -->
+        <fileSet>
+            <directory>../samples/CarbonSaaSTest</directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/samples/CarbonSaas</outputDirectory>

```

Comment:
The Output directory name is different to the name in the samples/pom.xml. This will break the build when trying to build the samples in the binary pack.